packages/internal/src/__tests__/build_api.test.ts

```diff
@@ -93,9 +95,195 @@ test.skip('api prebuild transforms gql with `babel-plugin-graphql-tag`', () => {
+  )
+  expect(code).toContain(
+    `import _Math$scale from "@babel/runtime-corejs3/core-js/math/scale";`
+  )
+  expect(code).toContain(
+    `import _Math$seededPRNG from "@babel/runtime-corejs3/core-js/math/seeded-prng";`
+  )
+  expect(code).toContain(
+    `import _Math$signbit from "@babel/runtime-corejs3/core-js/math/signbit";`
+  )
+  expect(code).toContain(
+    `import _Math$iaddh from "@babel/runtime-corejs3/core-js/math/iaddh";`
+  )
+  expect(code).toContain(
+    `import _Math$imulh from "@babel/runtime-corejs3/core-js/math/imulh";`
+  )
+  expect(code).toContain(
+    `import _Math$isubh from "@babel/runtime-corejs3/core-js/math/isubh";`
+  )
+  expect(code).toContain(
+    `import _Math$umulh from "@babel/runtime-corejs3/core-js/math/umulh";`
+  )
+
+  expect(code).toContain(
+    `import _Number$fromString from "@babel/runtime-corejs3/core-js/number/from-string";`
+  )
+
   expect(code).toContain(
-    'import _replaceAllInstanceProperty from "@babel/runtime-corejs3/core-js/instance/replace-all"'
+    `import _Observable from "@babel/runtime-corejs3/core-js/observable";`
+  )
+  expect(code).toContain(
+    `import _Symbol$observable from "@babel/runtime-corejs3/core-js/symbol/observable";`
+  )
+
+  expect(code).toContain(
+    `import _Promise from "@babel/runtime-corejs3/core-js/promise";`
+  )
+  const _Promise = require('@babel/runtime-corejs3/core-js/promise')
+  expect(_Promise).toHaveProperty('any')
+  expect(_Promise).toHaveProperty('try')
+
+  expect(code).toContain(
+    `import _Reflect$defineMetadata from "@babel/runtime-corejs3/core-js/reflect/define-metadata";`
+  )
+  expect(code).toContain(
+    `import _Reflect$ownKeys from "@babel/runtime-corejs3/core-js/reflect/own-keys";`
+  )
+  expect(code).toContain(
+    `import _Reflect$getOwnMetadataKeys from "@babel/runtime-corejs3/core-js/reflect/get-own-metadata-keys";`
+  )
+  expect(code).toContain(
+    `import _Reflect$getOwnMetadata from "@babel/runtime-corejs3/core-js/reflect/get-own-metadata";`
+  )
+
+  expect(code).toContain(
+    `import _Set from "@babel/runtime-corejs3/core-js/set";`
+  )
+  const _Set = require('@babel/runtime-corejs3/core-js/set')
+  expect(_Set).toHaveProperty('addAll')
+  expect(_Set).toHaveProperty('deleteAll')
+  expect(_Set).toHaveProperty('difference')
+  expect(_Set).toHaveProperty('every')
+  expect(_Set).toHaveProperty('filter')
+  expect(_Set).toHaveProperty('find')
+  expect(_Set).toHaveProperty('from')
+  expect(_Set).toHaveProperty('intersection')
+  expect(_Set).toHaveProperty('isDisjointFrom')
+  expect(_Set).toHaveProperty('isSubsetOf')
+  expect(_Set).toHaveProperty('isSupersetOf')
+  expect(_Set).toHaveProperty('join')
+  expect(_Set).toHaveProperty('map')
+  expect(_Set).toHaveProperty('of')
+  expect(_Set).toHaveProperty('reduce')
+  expect(_Set).toHaveProperty('some')
+  expect(_Set).toHaveProperty('symmetricDifference')
+  expect(_Set).toHaveProperty('union')
```

Comment:
For some things like `Map`, `Promise`, `Set`, etc, Babel doesn't polyfill the individual methods, but the whole object. For other things it does the opposite (see `Reflect`).